File: ambeth/Ambeth.Merge/ambeth/orm/IOrmXmlReader.cs
using System.Collections.Generic;
using System.Xml.Linq;

namespace De.Osthus.Ambeth.Orm
{
    public interface IOrmXmlReader
    {
		ISet<IEntityConfig> LoadFromDocument(XDocument doc);

		void LoadFromDocument(XDocument doc, ISet<IEntityConfig> localEntities, ISet<IEntityConfig> externalEntities);
    }
}

File: jambeth/jAmbeth.Merge/src/main/java/de/osthus/ambeth/orm/IOrmXmlReader.java
package de.osthus.ambeth.orm;

import java.util.Set;

import org.w3c.dom.Document;

public interface IOrmXmlReader
{
	Set<EntityConfig> loadFromDocument(Document doc);

	void loadFromDocument(Document doc, Set<EntityConfig> localEntities, Set<EntityConfig> externalEntities);
}